.github/workflows/containers-and-az-pool.yaml

```diff
@@ -90,16 +89,16 @@ jobs:
             ${{ env.REGISTRY}}/${{ env.IMAGE_NAME }}-dependencies:${{ steps.image-tag.outputs.tag }}
           file: ./Dockerfile-dependencies
 
-  _01_build-model-image:
+  build-pipeline-image:
```

Comment:
Why drop the numbers from the jobs within a workflow given that these do run sequentially?

Reply:
Sequentially is reflected in the dependency graph. What are you labeling jobs that concurrently run, but both depend on a third job? 1a and 1b? I guess we could do that, but I think this is cleaner. Nonetheless, I don't have a strong opinion about this, so I could rename the jobs to be 1-N.

Reply:
Nope that makes sense.